runtime/js/12_io.js

```diff
@@ -183,6 +172,44 @@
     return contents;
   }
 
+  function readAllSyncSized(r, size) {
+    const buf = new Uint8Array(size);
+    let cursor = 0;
+
+    while (cursor < size) {
+      const sliceEnd = MathMin(size, cursor + READ_PER_ITER);
+      const slice = buf.subarray(cursor, sliceEnd);
+      const read = r.readSync(slice);
+      if (typeof read == "number") {
+        cursor += read;
+      } else {
+        break;
+      }
+    }
+
+    return buf;
+  }
+
+  async function readAllInnerSized(r, size, options) {
```

Comment:
`readAll`/`writeAll` have been deprecated in favor or version in `deno_std`. Maybe better to add them there instead?

Reply:
This PR impacts `Deno.readFile` and `Deno.readFileSync` not `Deno.readAll`